api/src/pcapi/routes/backoffice/finance/blueprint.py

```diff
@@ -323,7 +324,7 @@ def create_individual_booking_incident() -> utils.BackofficeResponse:
 def create_collective_booking_incident(collective_booking_id: int) -> utils.BackofficeResponse:
     form = forms.IncidentCreationForm()
 
-    if not form.validate():
+    if not form.validate() or not form.total_amount.data:
```

Comment:
[question] Pourquoi cet ajout : `validate()` devrait déjà avoir vérifié le contenu de ce champ, non ?

Reply:
Car dans certains cas, on grise le champ total_amount (si plusieurs réservations sont sélectionnées par exemple pour le moment on n'autorise que le trop perçu total). Et lorsque le champ est disabled, la data n'est pas inclue dans les data du formulaire à son submit. On a donc dû passer ce champ en optionnel.

La problématique était, que se passe t-il dans le cas d'un trop percu partiel sur plusieurs réservation venant d'offres différentes, ou de stocks différents ? Aucune possibilité de déduire le montant revalorisé pour chaque réservation sélectionnée. Dans un futur proche on va sans doute limiter la sélection aux bookings du même stock mais nous l'avons mis de côté pour le moment pour se concentrer sur les cas d'incidents plus fréquents

Reply:
Erreur de ma part, je n'ai pas en tête la raison de cet ajout mais après réflexion effectivement ca ne sert à rien :D

Reply:
Cela vient du fait que le champ est optionnel (car si plusieurs réservations sont sélectionnées pour le moment on ne se limite qu'à du trop percu TOTAL). Du fait qu'il soit fonctionnel, la donnée n'est pas récupérée à la validation du formulaire. Ce qui est problématique si une seule réservation est sélectionnée pour du trop percu partiel puisque que l'on veut récupérer la donnée. Je devrais donc modifier la condition comme ceci : 
`... or (len(bookings) == 1 and not form.total_amount.data)`

La problématique que l'on avait avec le trop perçu partiel sur plusieurs réservation c'était comment savoir quel nouveau montant attribuer à chaque réservation, leurs montants pouvant varier d'une résa à une autre. On a décidé avec Solène de se concentrer sur les cas vraiment urgents et de se repencher dessus plus tard